docs/BondingCurve.md

```diff
@@ -0,0 +1,170 @@
+# Bonding Curve Fundraising
+
+The bonding curve fundraising module enables projects and organizations to issue tokens and raise money to fund their vision. The core of this implementation is a **bonding curve**, or automated market maker - as conceptualized by individuals such as [Simon dela Rouviere](https://medium.com/@simondlr/tokens-2-0-curved-token-bonding-in-curation-markets-1764a2e0bee5), [Billy Rennekamp](https://medium.com/@billyrennekamp/converting-between-bancor-and-bonding-curve-price-formulas-9c11309062f5) and [Thibauld Favre](https://github.com/C-ORG/whitepaper) to enable continuous funding for organizations, coupled with guaranteed liquidity for investors without relying on exchanges.
+
+Tokens issued via bonding curves can offer rights in the organization, such as **dividends** on future revenue or **governance rights** through token locking.
```

Comment:
". . . through token locking." . . . I find this a little confusing . . do these dividends or governance rights require token locking?  Thinking we could drop that phrase from the sentence.